lib/terrafying/components/dynamicset.rb

```diff
@@ -162,7 +162,7 @@ def generate_template(health_check, instances, launch_config, subnets,tags, roll
                 ],
                 MinSize: "#{instances[:min]}",
                 DesiredCapacity: "#{instances[:desired]}",
-                Tags: tags,
+                Tags: instances[:tags].merge(tags),
```

Comment:
you've left some of the instance tags commit in

Reply:
removed